pyanaconda/modules/subscription/__main__.py

```diff
@@ -17,9 +17,16 @@
 # License and may only be used or replicated with the express permission of
 # Red Hat, Inc.
 #
+
+# Initialize the service.
 from pyanaconda.modules.common import init
 init()
 
+# Check the initial conditions.
+from pyanaconda.modules.subscription.initialization import check_initial_conditions
+check_initial_conditions()
```

Comment:
I know you probably handled this correctly but I want to check. Are we sure that this won't stay in a loop when something will start the addon again and again by dbus/systemd-service and it would kill itself all the time?